docs/en/ingest-management/commands.asciidoc

```diff
@@ -25,6 +26,33 @@ run these commands.
 <hr>
 ++++
 
+[discrete]
+[[elastic-agent-status-command]]
+== elastic-agent status
```

Comment:
Do you mind moving this section? We like to show the commands sorted alphabetically.